calculator.js

```diff
@@ -238,7 +238,7 @@ $( document ).ready(function() {
     //   - Change "theme name" to a catchy name (can be your name)
     //   - Change the colors blue and red with colors of your choice.
     //   - Uncomment the call to addOption (remove the `//`)
-    // addOption("theme name", "linear-gradient(blue, red)", "red", "red");
+    addOption("Wolf theme", "linear-gradient(blue, orange)", "blue", "white");
```

Comment:
I have bad tastes in colors... here are my bad taste suggestions!

```suggestion
    addOption("Wolf theme", "linear-gradient(green, pink)", "blue", "white");
```